lib/src/react_client/event_helpers.dart

```diff
@@ -271,7 +425,7 @@ SyntheticMouseEvent createSyntheticMouseEvent({
   num clientX,
   num clientY,
   bool ctrlKey,
-  SyntheticDataTransfer dataTransfer,
+  dynamic dataTransfer,
```

Comment:
Why was this changed to `dynamic`?

Reply:
Because if a consumer passes in a `dataTransfer` from an event created by the browser, the instance may be a `DataTransfer` instance, or they could pass in an instance that is a `SyntheticDataTransfer` if the event has already been processed somehow